exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -118,22 +120,55 @@ def get_resource_data(
     return _get_resource_data(sdk_resource_scope_data, resource_class, name)
 
 
-def _load_credential_from_file(filepath) -> ChannelCredentials:
+def _read_file(file_path: Optional[str]) -> Optional[bytes]:
+    if file_path is None:
+        logger.error(
+            "No file path provided for reading. Please check the environment variables."
+        )
+        return None
+
     try:
-        with open(filepath, "rb") as creds_file:
-            credential = creds_file.read()
-            return ssl_channel_credentials(credential)
-    except FileNotFoundError:
-        logger.exception("Failed to read credential file")
+        with open(file_path, "rb") as file:
+            return file.read()
+    except FileNotFoundError as e:
+        logger.exception(
+            f"Failed to read file: {e.filename}. Please check if the file exists and is accessible."
+        )
         return None
 
 
-def _get_credentials(creds, environ_key):
+def _load_credentials(
+    certificate_file: str,
```

Comment:
I think these should be `Optional[str]` and only perform `_read_file` if params are `None` so we only perform this if user explicitly configured it.

Reply:
Good, point, fixed it :) @lzchen